moped-editor/src/queries/components.js

```diff
@@ -44,10 +48,6 @@ export const GET_PROJECT_COMPONENTS = gql`
           internal_table
         }
         line_representation
-        moped_subcomponents {
-          subcomponent_id
-          subcomponent_name
-        }
```

Comment:
this was kinda tripping me up, b/c we were sourcing the subcomponent options from either the proj_component or the lookup table depending on edit state. so the changes to `useSubcomponentOptions` signature below are following from this change to always just source the allowed subcomponents from the lookup table data.

Reply:
Thanks for catching this! 🙏